modules/audiencerunBidAdapter.js

```diff
@@ -0,0 +1,143 @@
+import * as utils from '../src/utils';
+import { config } from '../src/config';
+import { registerBidder } from '../src/adapters/bidderFactory';
+import { BANNER } from '../src/mediaTypes';
+
+const BIDDER_CODE = 'audiencerun';
+const ENDPOINT_URL = 'https://d.audiencerun.com/prebid';
+
+export const spec = {
+  version: '1.0.0',
+  code: BIDDER_CODE,
+  supportedMediaTypes: [BANNER],
+
+  /**
+   * Determines whether or not the given bid request is valid.
+   *
+   * @param {object} bid The bid to validate.
+   * @return boolean True if this is a valid bid, and false otherwise.
+   */
+  isBidRequestValid: function (bid) {
+    let isValid = true;
+    if (!utils.deepAccess(bid, 'params.zoneId')) {
+      utils.logError('AudienceRun zoneId parameter is required. Bid aborted.');
+      isValid = false;
+    }
+    return isValid;
+  },
+
+  /**
+   * Make a server request from the list of BidRequests.
+   *
+   * @param {BidRequest[]} bidRequests A non-empty list of bid requests which should be sent to the Server.
+   * @return ServerRequest Info describing the request to the server.
+   */
+  buildRequests: function(bidRequests, bidderRequest) {
+    const bids = bidRequests.map(bid => {
+      return {
+        zoneId: utils.getValue(bid.params, 'zoneId'),
+        sizes: bid.sizes.map(size => ({
+          w: size[0],
+          h: size[1]
+        })),
+        bidfloor: bid.params.bidfloor || 0.0,
+        bidId: bid.bidId,
+        bidderRequestId: utils.getBidIdParameter('bidderRequestId', bid),
+        adUnitCode: utils.getBidIdParameter('adUnitCode', bid),
+        auctionId: utils.getBidIdParameter('auctionId', bid),
+        transactionId: utils.getBidIdParameter('transactionId', bid)
+      };
+    });
+
+    const payload = {
+      libVersion: this.version,
+      referer: bidderRequest.refererInfo ? bidderRequest.refererInfo.referer || null : null,
+      currencyCode: config.getConfig('currency.adServerCurrency'),
+      timeout: config.getConfig('bidderTimeout'),
+      bids
+    };
+
+    if (bidderRequest && bidderRequest.gdprConsent) {
```

Comment:
Will you add support for CCPA or will you only support GDPR?

Reply:
not immediately